server/sonar-web/src/main/js/apps/organizations/components/OrganizationEdit.js

```diff
@@ -68,17 +71,17 @@ class OrganizationEdit extends React.PureComponent {
     this.mounted = false;
   }
 
-  handleAvatarInputChange = (e: Object) => {
+  handleAvatarInputChange = (e /*: Object */) => {
     const { value } = e.target;
     this.setState({ avatar: value });
     this.changeAvatarImage(value);
   };
 
-  changeAvatarImage = (value: string) => {
+  changeAvatarImage = (value /*: string */) => {
     this.setState({ avatarImage: value });
   };
 
-  handleSubmit = (e: Object) => {
+  handleSubmit = (e /*: Object */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)